run-uploader.sh

```diff
@@ -62,6 +62,7 @@ convert_disk_img() {
 
   zcat "$TEMP_DISK_PATH" | dd conv=sparse of="${TEMP_DISK_PATH%.gz}"
   qemu-img convert -f raw -O qcow2 "${TEMP_DISK_PATH%.gz}" $CONVERTED_DISK_PATH
```

Comment:
I have to grab the params from CDI but I believe `-t=none` is desired here (bypass host page cache, write directly to disk). There's also `-p` for progress

Reply:
Interesting. I'll check it.